src/session.cpp

```diff
@@ -50,22 +55,41 @@ void Session::close(OnCompleteCallback onClosedCallback)
                                       (role == boost::beast::role_type::server) ? "server" : "client";
                                   if (wsStream->is_open())
                                   {
-                                      NS_LOG_W("Closing {} session failure: {}", roleName, ec.message());
+                                      NS_LOG_E("Closing {} session failure: {}", roleName, ec.message());
                                       onClosedCallback(ec);
                                   }
                                   else
                                   {
-                                      NS_LOG_T("Closed {} session", roleName);
+                                      NS_LOG_I("{} session successfully closed", roleName);
```

Comment:
Similarly here, I'd probably keep session information as debug, but have a more understandable message logged as info in the SDK (eg. device successfully disconnected).